src/com/mitrais/view/WelcomeScreen.java

```diff
@@ -0,0 +1,49 @@
+package src.com.mitrais.view;
+
+import src.com.mitrais.model.Account;
+import src.com.mitrais.utils.Helper;
+import src.com.mitrais.utils.Validation;
+
+import java.util.List;
+import java.util.Scanner;
+
+public class WelcomeScreen extends BaseScreen {
+    private Scanner in = new Scanner(System.in);
+    private TransactionScreen transactionScreen = new TransactionScreen();
+    Validation validation = new Validation();
+
+    public void show(){
+        System.out.println("======================= ATM Simulation =======================");
+        System.out.println("Enter Account Number: ");
+
+        String accountNumber = in.nextLine();
+
+        List<String> errAccount = validation.ValidateAccountNumberOrPin(accountNumber, true);
+        if(!errAccount.isEmpty()){
+            for (String error : errAccount) {
+                System.out.println(">> " + error);
+                return;
+            }
+        }
+
+        System.out.println("Enter PIN: ");
+        String pin = in.nextLine();
+
+        errAccount = validation.ValidateAccountNumberOrPin(pin, false);
+        if(!errAccount.isEmpty()){
+            for (String error : errAccount) {
+                System.out.println(">> " + error);
+                return;
+            }
+        }
+        Account account = accountDao.getLoginAccount(accountNumber, pin);
+        if(account == null){
+            Helper.resetActiveAccount();
```

Comment:
If you perform any transaction and then you exit and re-enter (without stopping it), your app does not remember the changes. Try for example withdrawing 10 from the first account, then exit and re enter, you'll see the balance is reset. Consider it